lib/Yield/Pipes/CallGraphPipes.cpp

```diff
@@ -35,7 +35,10 @@ void ProcessCallGraph::run(pipeline::Context &Context,
   SortedVector<efa::FunctionMetadata> Metadata;
   for (const auto &LLVMFunction : FunctionTags::Isolated.functions(&Module))
     Metadata.insert(*::detail::extractFunctionMetadata(&LLVMFunction));
-  revng_assert(Metadata.size() == Model->Functions().size());
+
+  // If some functions are missing, do not output anything
+  if (Metadata.size() != Model->Functions().size())
+    return;
```

Comment:
Uhm, this doesn't seem right.

In order to produce the call graph we clearly need all of the functions or fail.

Reply:
It's simply checking if the number of metadata entries we have is the same as the number of functions; if not – the call graph is likely to end up broken – we fail. I remember we were previously hard-asserting this, but some issue either with the pipeline or the build tests prevented us from doing it.